src/settings-ui/Settings.UI/SettingsXAML/Views/ShellPage.xaml

```diff
@@ -105,135 +105,158 @@
+                    <NavigationViewItem.MenuItems>
+                        <NavigationViewItem
+                            x:Uid="Shell_PowerPreview"
+                            helpers:NavHelper.NavigateTo="views:PowerPreviewPage"
+                            Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/FileExplorerPreview.png}" />
+                        <!--  File Explorer Thumbnails  -->
+                        <NavigationViewItem
+                            x:Uid="Shell_FileLocksmith"
+                            helpers:NavHelper.NavigateTo="views:FileLocksmithPage"
+                            Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/FileLocksmith.png}" />
+                        <NavigationViewItem
+                            x:Uid="Shell_ImageResizer"
+                            helpers:NavHelper.NavigateTo="views:ImageResizerPage"
+                            Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/ImageResizer.png}" />
+                        <NavigationViewItem
+                            x:Uid="NewPlus_Product_Name"
+                            helpers:NavHelper.NavigateTo="views:NewPlusPage"
+                            Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/NewPlus.png}" />
+                        <NavigationViewItem
+                            x:Uid="Shell_Peek"
+                            helpers:NavHelper.NavigateTo="views:PeekPage"
+                            Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/Peek.png}" />
+                        <NavigationViewItem
+                            x:Uid="Shell_PowerRename"
+                            helpers:NavHelper.NavigateTo="views:PowerRenamePage"
+                            Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/PowerRename.png}" />
+                    </NavigationViewItem.MenuItems>
+                </NavigationViewItem>
+                <!--  Advanced  -->
                 <NavigationViewItem
-                    x:Uid="Shell_Workspaces"
-                    helpers:NavHelper.NavigateTo="views:WorkspacesPage"
-                    Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/Workspaces.png}" />
+                    x:Uid="Shell_TopLevelAdvanced"
+                    Icon="Page2"
+                    SelectsOnInvoked="False">
+                    <NavigationViewItem.MenuItems>
+                        <NavigationViewItem
+                            x:Uid="Shell_CmdNotFound"
+                            helpers:NavHelper.NavigateTo="views:CmdNotFoundPage"
+                            Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/CommandNotFound.png}" />
+                        <NavigationViewItem
+                            x:Uid="Shell_EnvironmentVariables"
+                            helpers:NavHelper.NavigateTo="views:EnvironmentVariablesPage"
+                            Icon="{ui:BitmapIcon Source=/Assets/Settings/Icons/EnvironmentVariables.png}" />
```

Comment:
Feels more like a (os) system tool. What makes the difference between advanced and system tools?